src/utils/appData.spec.ts

```diff
@@ -112,3 +114,31 @@ test('Valid IPFS appData from CID', async () => {
 
   expect(validation).toEqual(VALID_RESULT)
 })
+
+test('Valid: quote metadata - minimal', async () => {
+  const document = {...BASE_DOCUMENT, metadata: {quote: {sellAmount: '1', buyAmount: '1'}}}
+  const validation = await validateAppDataDocument(document)
+
+  expect(validation).toEqual(VALID_RESULT)
+})
+
+test('Valid: quote metadata - with all fields', async () => {
+  const document = {...BASE_DOCUMENT, metadata: {quote: {sellAmount: '1', buyAmount: '1', id: 'S09D8ZFAX'}}}
+  const validation = await validateAppDataDocument(document)
+
+  expect(validation).toEqual(VALID_RESULT)
+})
+
+test('Invalid: quote metadata - missing fields', async () => {
+  const document = {...BASE_DOCUMENT, metadata: {quote: {}}}
+  const validation = await validateAppDataDocument(document)
+
+  expect(validation.result).toBeFalsy()
+})
+
+test('Invalid: quote metadata - wrong type', async () => {
+  const document = {...BASE_DOCUMENT, metadata: {quote: {sellAmount: 312, buyAmount: '0xbab3'}}}
```

Comment:
is it possible that prettier hasn't processed this file?

Reply:
Yes, thanks for noticing it!
I did not have prettier setup in my IDE.
Fixed now